src/Microsoft.OData.ModelBuilder/Microsoft.OData.ModelBuilder.xml

```diff
@@ -5106,5 +5116,2153 @@
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration.HasCollectionProperty(Microsoft.OData.Edm.Vocabularies.EdmPropertyPathExpression)">
+            <summary>
+            Restricted Collection-valued property
+            </summary>
+            <param name="collectionProperty">The value to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration.AddFilterFunctions(System.String[])">
+            <summary>
+            List of functions and operators supported in filter expressions.
+            If not specified, null, or empty, all functions and operators may be attempted.
+            </summary>
+            <param name="filterFunctions">The value(s) to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration.HasFilterRestrictions(Microsoft.OData.ModelBuilder.Capabilities.V1.FilterRestrictionsConfiguration)">
+            <summary>
+            Restrictions on filter expressions
+            </summary>
+            <param name="filterRestrictions">The value to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration.HasSearchRestrictions(Microsoft.OData.ModelBuilder.Capabilities.V1.SearchRestrictionsConfiguration)">
+            <summary>
+            Restrictions on search expressions
+            </summary>
+            <param name="searchRestrictions">The value to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration.HasSortRestrictions(Microsoft.OData.ModelBuilder.Capabilities.V1.SortRestrictionsConfiguration)">
+            <summary>
+            Restrictions on orderby expressions
+            </summary>
+            <param name="sortRestrictions">The value to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration.IsTopSupported(System.Boolean)">
+            <summary>
+            Supports $top
```

Comment:
```suggestion
            Supports $top query option
```

Reply:
Auto generated, could be fixed at [source](https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Capabilities.V1.xml)...